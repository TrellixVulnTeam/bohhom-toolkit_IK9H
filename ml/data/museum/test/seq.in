Hello robot
Excuse me
How should I address you
What should I call you
can I have your name
could you explain this painting
I would like to know more about this painting
can you introduce this item
i want your language changed to chinese
can you start talking to me in spanish from here on out
please speak to me in chinese
please make your language setting french
please speak german to me from now on
i need your language setting to be russian now
please change your language setting to spanish
i want your responses to be in french
from now on, i want you to answer me using japanese
how should I go to the European Art room
I would like to go to the European Art exhibit
Can you take me to the European Art exhibt please
can you take me to showroom B
Salute
show me where the food court is please
