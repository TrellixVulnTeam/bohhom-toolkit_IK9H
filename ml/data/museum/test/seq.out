O O
O O
O O O O O
O O O O O
O O O O O
O O O O O
O O O O O O O O O
O O O O O
O O O O O O B-language
O O O O O O O B-language O O O O
O O O O O B-language
O O O O O B-language
O O B-language O O O O O
O O O O O O O B-language O
O O O O O O B-language
O O O O O O O B-language
O O O O O O O O O O B-language
O O O O O O B-location I-location I-location
O O O O O O O B-location I-location I-location
O O O O O O B-location I-location I-location O
O O O O O B-location I-location
O
O O O O B-location I-location O O
O B-position
O B-position
O B-position
O B-position
O B-position
